README.md

```diff
@@ -0,0 +1,91 @@
+# UrContacts
+> A contact list application
+
+UrContacts is a React contact list app built with full CRUD capabilities. It uses json-server to locally host a JSON file database on the backend, where your data will persist.
+
+## Installation and Build
+
+1. `npm init -y`
+
+2. This app was built with Vite. While you may use whatever React bootstrapping tool you prefer to get the app running, Vite has some useful features built-in, such as TypeScript and a server: [Vite][vite]. Instructions from here on out will assume you have some way a spinning a server locally.
+
+3. If you use a bootstrapping tool other than Vite, make sure it has TypeScript baked in, or install TypeScript separately: [TypeScript][typescript]
+
+4. `npm install --save json-server react-router-dom` 
+
+5. Add your server start scripts to your `package.json`:
+
+    "scripts": {
+        // .. 
+        "server:dev": "json-server --watch ./data/db.json",
+        "start": "json-server ./data/db.json --static ./dist"
+        //..
+    }
+
+6. To run your server: `npm run server:dev`
+
+7. In a separate terminal, start your React app using Vite: `npm run dev`
```

Comment:
For end-user, you'll want to create a separate start script that is not `dev` (dev = you, basically)

They should be able to just call `npm run build` and then `npm start` (which if you check the `package.json`, we've already added the right commands there for starting both server & serving the client)

Reply:
Yes! Makes sense. Question: the script says `build`, but does it still need `run` before it when they use npm. In other words, they need to enter `npm run build` instead of `npm build`?